### PR TITLE
Add packaging to minimal with test

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Found during the data conversion workshop, specific to NeuroConv, but noticed it might also apply here too

### CODE REVIEW

**Review by h-mayorquin — APPROVED** [review]
LGTM

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -33,6 +33,11 @@ jobs:
 
       - name: Install roiextractors with minimal requirements
         run: pip install .[test]
+      - name: Test initial import of all non-lazy dependencies
+        run: python -c "import roiextractors"
+
+      - name: Install roiextractors with testing requirements
```
[the remark this answers is not in the document]
Comment (CodyCBakerPhD) [reply .github/workflows/run-tests.yml:39]: The first time should have only been purely minimal (no extras)

Every other step adds more and more extras

**Comment by CodyCBakerPhD** [comment]
Failures in https://github.com/catalystneuro/roiextractors/actions/runs/9603499178/job/26486881927 evidence that we do indeed need packaging in minimal these days

**Comment by pauladkisson** [comment]
Can you add this PR to the changelog as well?

**Review by pauladkisson — APPROVED** [review]

**Pull request merged**